examples/series-canvas-autobandwidth/README.md

```diff
@@ -0,0 +1,3 @@
+# Series Canvas Autobandwidth
```

Comment:
Should the folder be called series-canvas-auto-bandwidth? And this be Series Canvas Auto Bandwidth?